client/src/main/com/sinch/sdk/domains/voice/adapters/converters/CalloutsDtoConverter.java

```diff
@@ -0,0 +1,128 @@
+      CalloutRequestParametersCustom parameters = (CalloutRequestParametersCustom) client;
+      return convert(parameters);
+    }
+    return null;
+  }
+
+  public static String convert(GetCalloutResponseObjDto dto) {
+    if (null == dto) {
+      return null;
+    }
+    return dto.getCallId();
+  }
+
+  private static CalloutRequestDto convert(CalloutRequestParametersConference client) {
+
+    ConferenceCalloutRequestDto dto =
+        new ConferenceCalloutRequestDto().destination(convert(client.getDestination()));
+
+    client.getCli().ifPresent(f -> dto.setCli(f.stringValue()));
+    client.getDtfm().ifPresent(dto::setDtmf);
+    client.getCustom().ifPresent(dto::setCustom);
+
+    dto.conferenceId(client.getConferenceId());
+    client.getDtfmOptions().ifPresent(f -> dto.setConferenceDtmfOptions(convert(f)));
+
+    client.getMaxDuration().ifPresent(dto::setMaxDuration);
+    client.getEnableAce().ifPresent(dto::setEnableAce);
+    client.getEnableDice().ifPresent(dto::setEnableDice);
+    client.getEnablePie().ifPresent(dto::setEnablePie);
+    client.getLocale().ifPresent(dto::setLocale);
+    client.getGreeting().ifPresent(dto::setGreeting);
+    client.getMohClass().ifPresent(f -> dto.setMohClass(f.value()));
+    client.getDomain().ifPresent(f -> dto.setDomain(f.value()));
+
+    return new CalloutRequestDto()
+        .method(MethodEnum.CONFERENCECALLOUT.getValue())
+        .conferenceCallout(dto);
+  }
+
+  private static CalloutRequestDto convert(CalloutRequestParametersTTS client) {
+
+    TtsCalloutRequestDto dto =
+        new TtsCalloutRequestDto().destination(convert(client.getDestination()));
+
+    client.getCli().ifPresent(f -> dto.setCli(f.stringValue()));
+    client.getDtfm().ifPresent(dto::setDtmf);
+    client.getCustom().ifPresent(dto::setCustom);
+    client.getEnableAce().ifPresent(dto::setEnableAce);
+    client.getEnableDice().ifPresent(dto::setEnableDice);
+    client.getEnablePie().ifPresent(dto::setEnablePie);
+    client.getLocale().ifPresent(dto::setLocale);
+    client.getDomain().ifPresent(f -> dto.setDomain(f.value()));
+    client.getText().ifPresent(dto::setText);
+    client.getPrompts().ifPresent(dto::setPrompts);
+
+    return new CalloutRequestDto().method(MethodEnum.TTSCALLOUT.getValue()).ttsCallout(dto);
+  }
+
+  private static CalloutRequestDto convert(CalloutRequestParametersCustom client) {
+
+    CustomCalloutRequestDto dto =
+        new CustomCalloutRequestDto().destination(convert(client.getDestination()));
+
+    client.getCli().ifPresent(f -> dto.setCli(f.stringValue()));
+    client.getDtfm().ifPresent(dto::setDtmf);
+    client.getCustom().ifPresent(dto::setCustom);
+
+    client.getMaxDuration().ifPresent(dto::setMaxDuration);
+    client.getIce().ifPresent(dto::setIce);
+    client.getAce().ifPresent(dto::setAce);
+    client.getPie().ifPresent(dto::setPie);
+
+    return new CalloutRequestDto().method(MethodEnum.CUSTOMCALLOUT.getValue()).customCallout(dto);
```

Comment:
And for `CUSTOMCALLOUT`

Reply:
_Same answer as for CONFERENCE_CALLOUT_

Reply:
/me drying my tears and facing the hard truth